tests/DocTest_Tests/reporters/approval_tests/CommandLineReporterTests.Test_Command_Lines.approved.txt

```diff
@@ -66,6 +70,10 @@ windows: start "" "{ProgramFiles}Devart\Code Compare\CodeCompare.exe" "a.txt" "b
 unix   : "{ProgramFiles}Devart\Code Compare\CodeCompare.exe" "a.txt" "b.txt" &
 cygwin : "$(cygpath '{ProgramFiles}Devart\Code Compare\CodeCompare.exe')" "$(cygpath -aw 'a.txt')" "$(cygpath -aw 'b.txt')" &
 
+windows: cmd /S /C ""{ProgramFiles}Sublime Merge\smerge.exe" mergetool --no-wait "a.txt" "b.txt" -o "b.txt""
```

Comment:
Whilst running the tests on my machine, I just noticed the double quotes at the start and end of this line... These are the first reporters in this file to run in the foreground, so it's the first time I've seen this output...

Naively, I would have thought that the pairs of double-quotes would have confused the CMD exe - but as you've tested it on Windows, I guess it somehow just magically works....

Reply:
I think I may have got the quotes wrong on this, for the foreground case...

https://github.com/approvals/ApprovalTests.cpp/blob/75f65da3a2dcf03c5fda529a374c1e81384ea3c1/ApprovalTests/launchers/SystemLauncher.h#L64-L65

Reply:
I've got to go out now... I'll look at it on a PC later...

Reply:
It definitely didn't cause any issues in Windows, so that's good.